eng/pipelines/mgmt.yml

```diff
@@ -0,0 +1,43 @@
+# External variables Settable at queue time:
+# "pomFile" : "pom.xml".   This is the project object model file for Maven.
+
+trigger:
+  - master
+
+variables:
+  DefaultOptions: '--batch-mode -Dmaven.wagon.http.pool=false'
```

Comment:
These are no longer being used now that you are using the script.

Reply:
moved these to the script